app/internal_packages/preferences/lib/tabs/keymaps/displayed-keybindings.js

```diff
@@ -31,6 +31,7 @@ module.exports = [
       ['composer:focus-to', 'Focus the To field'],
       ['composer:show-and-focus-cc', 'Focus the Cc field'],
       ['composer:show-and-focus-bcc', 'Focus the Bcc field'],
+      ['composer:select-attachment', 'Select file attachment'],
```

Comment:
I'm not sure what style this project conforms to, but this hanging `,` isnt needed.

Reply:
Hanging commas are throughout the project, so I've kept them for consistency. I'm assuming Airbnb-based style guide?

Reply:
ah, I must have missed that. When I glanced quickly I thought I didnt see them. You probably looked more thoroughly.